README.md

```diff
@@ -90,8 +90,11 @@ Some commands you'll want to learn and what they do:
   * Displays all levels in the specified skill
 * git gud load <skill_name>
   * Begin the first level of the specified skill
+* git gud load -<level_name>
+  * Begin the specified level of the current skill
 * git gud load <skill_name> <level_name>
-  * Begin the specified level (it is optional to specify the level)
+  * Begin the specified level (specifying the level is optional)
+  * A hyphen can be used instead of a space between <skill_name> and <level_name> (specifying the skill is optional)

```

Comment:
Don't say that specifying the skill is optional. We already have the other thing above